2.4/root/usr/bin/run-mongod

```diff
@@ -74,26 +74,23 @@ if [ -z "${MONGODB_REPLICA_NAME-}" ]; then
     usage
   fi
   # Run the MongoDB in 'standalone' mode
-  if [ ! -f ${MONGODB_DATADIR}/.mongodb_datadir_initialized ]; then
-    # Create MongoDB users and restart MongoDB with authentication enabled
-    # At this time the MongoDB does not accept the incoming connections.
-    mongod $mongo_common_args & #--bind_ip 127.0.0.1 --quiet >/dev/null &
-    wait_for_mongo_up
+  mongod $mongo_common_args &
+  wait_for_mongo_up
+  if [ $(mongo admin --quiet --eval "db.system.users.find({'user':'admin'}).length()") == "1" ]; then
```

Comment:
@php-coder You are right, I've forgotten about this method. It is better...

Both should be fixed now.